DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml

```diff
@@ -1,10 +1,10 @@
 ﻿@using DigitalLearningSolutions.Web.Helpers
-@using DigitalLearningSolutions.Web.ViewModels.SignpostingSso
+@using DigitalLearningSolutions.Web.ViewModels.Signposting
 @model LinkLearningHubViewModel
 
 @{
-    ViewData[LayoutViewDataKeys.Title] = "Recommended Learning Resources - Account linked";
-    ViewData[LayoutViewDataKeys.Application] = "Recommended Learning Resources";
+    ViewData[LayoutViewDataKeys.Title] = "Learning Hub Account Linked";
+    ViewData[LayoutViewDataKeys.Application] = "Digital Learning Solutions Learning Portal";
     ViewData[LayoutViewDataKeys.HeaderPrefix] = "";

```

Comment:
I think you can just remove these lines and [SetDlsSubApplication] will achieve the same things.

Reply:
Nice, done.

Reply:
(You can skip TL review when peer-review is done.)